Makefile

```diff
@@ -1120,7 +1126,7 @@ ifdef LANGUAGES
 endif
 	$(BINDIST_CMD)
 
-export ODIR _OBJS LDFLAGS CXX W32FLAGS DEFINES CXXFLAGS TARGETSYSTEM CLANG PCH PCHFLAGS
+export ODIR _OBJS LDFLAGS CXX W32FLAGS DEFINES CXXFLAGS TARGETSYSTEM CLANG PCH PCHFLAGS PCH_SUFFIX
```

Comment:
Why did @ZhilkinSerg export _OBJS?

Reply:
Huh?

Reply:
It does not get to be used in `tests/Makefile`, seams useless to do it. Best case, worst it's confusing.

Reply:
How is it related to me?

Reply:
blame says:
`Zhilkin Serg 2020-12-28 01:11:26 +0300`
I'm sorry, if that's some how, wrong.